apps/web/app/dashboard/admin/page.tsx

```diff
@@ -275,14 +20,13 @@ export default function AdminPage() {
   }
 
   return (
-    <div className="flex flex-col gap-5 rounded-md border bg-background p-4">
-      <p className="text-2xl">Admin</p>
-      <Separator />
-      <ServerStatsSection />
-      <Separator />
-      <UsersSection />
-      <Separator />
-      <ActionsSection />
-    </div>
+    <>
+      <div className="rounded-md border bg-background p-4">
+        <ServerStats />
+      </div>
+      <div className="mt-4 rounded-md border bg-background p-4">
+        <Users />
```

Comment:
minor thing, but can we call this `UserList`?

Reply:
Fixed